### ISSUE TITLE
[EKS] Error: You must use a valid fully-formed launch template

### ISSUE THREAD

**Issue by mrichman** [aws/containers-roadmap#467]
I'm getting this issue deploying using `eksctl` as of today. I've made no changes to my deployment scripts:

> You must use a valid fully-formed launch template. EBS-optimized instances are not supported for your requested configuration. Please check the documentation for supported configurations. (Service: AmazonAutoScaling; Status Code: 400; Error Code: ValidationError; Request ID: a6251170-cfe8-11e9-ae88-234007dafb6a)

My `eksctl` command is:

```
AWS_REGION=us-east-1 /usr/local/bin/eksctl create cluster --region=us-east-1 --node-type=t2.medium --zones=us-east-1a,us-east-1b,us-east-1d --verbose=4 >> /var/log/eksctl.log 2>&1
```

My `eksctl` is pulled from https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_linux_amd64.tar.gz at deployment time.

**Comment by ejsmith** [comment]
I just got the same error.

**Comment by mrichman** [comment]
Looks like this will be resolved today: https://github.com/weaveworks/eksctl/issues/1293#issuecomment-528813215

**Comment by errordeveloper** [comment]
I can config this was resolved in eksctl 0.5.2, which was release just now.

**Comment by mrichman** [comment]
Confirming in my lab environment that eksctl 0.5.2 fixes the issues. The cluster provisioned without errors.
